client/src/partials/fiscal/fiscal.routes.js

```diff
@@ -11,19 +11,24 @@ angular.module('bhima.routes')
         controller: 'FiscalController as FiscalCtrl',
         templateUrl: 'partials/fiscal/fiscal.html'
       })
-        .state('fiscal.list', {
-          url : '',
-          templateUrl : 'partials/fiscal/fiscal.list.html'
-        })
-        .state('fiscal.create', {
-          url : '/create/new',
-          controller : 'FiscalCreateController as FiscalManageCtrl',
-          templateUrl : 'partials/fiscal/fiscal.manage.html'
-        })
-        .state('fiscal.update', {
-          url : '/update',
-          controller : 'FiscalUpdateController as FiscalManageCtrl',
-          templateUrl : 'partials/fiscal/fiscal.manage.html',
-          data : { label : null }
-        });
+      .state('fiscal.list', {
+        url : '',
+        templateUrl : 'partials/fiscal/fiscal.list.html'
+      })
+      .state('fiscal.create', {
+        url : '/create/new',
```

Comment:
Can this URL just be `/create`?

This might give you trouble.  The way to properly set up states is seen in `account.routes.js`.  You must:
 1. Remove `fiscal/:id` from the `fiscal` state.   Change the URL to `fiscal/`.
 2. The update state you use `:id/update` as the URL.
 3. And `:id/opening_balance` for the openingBalance state.